root/search/components/SearchForm.js

```diff
@@ -103,10 +103,6 @@ const SearchForm = ({form}: Props) => {
           </div>
         </form>
       </div>
-      <div className="searchinfo">
-        <strong>{l('Please note:')}</strong>
-        <p>{l('Search indexes are updated every 3 hours, use the direct database search if you require up to the minute correct results.')}</p>
-      </div>
```

Comment:
I would drop the strong `Please note:` but replace the paragraph with documentation:

``` javascript
        <p>
          {l('For more information, check the {doc_doc|documentation}.', {
            __react: true,
            doc_doc: '/doc/Search',
          })}
        </p>
```

It is already used in some entity edit forms, it is not an additional localization string.